src/startest/expect.gleam

```diff
@@ -132,6 +132,25 @@ pub fn string_to_contain(actual: String, expected: String) -> Nil {
   }
 }
 
+/// Asserts that the given string does not contain the expected string.
+pub fn string_not_to_contain(actual: String, expected: String) -> Nil {
```

Comment:
Could we call this `string_to_not_contain` instead?

```suggestion
pub fn string_to_not_contain(actual: String, expected: String) -> Nil {
```

That reads more how like English.

Reply:
Adjusted accordingly.